base-images-sbom-script/base_image_sbom_script.py

```diff
@@ -0,0 +1,86 @@
+import json
+import argparse
+import pathlib
+
+from packageurl import PackageURL
+
+
+def get_base_images_sbom_data(images, is_last_from_scratch):
+
+    new_data = {"formulation": [{"components": []}]}
+
+    for index, image in enumerate(images):
+        property_name = f"builder-{index}"
+        if index == len(images) - 1 and not is_last_from_scratch:
+           property_name = f"base-image"
+           
+           
+        repository, digest = image.split("@")
+        repository_without_tag, tag = repository.split(":")
+        # last fragment of the repository
+        name = repository_without_tag.split("/")[-1]
+        
+        purl = PackageURL(
+            type="oci",
+            name=name,
+            version=digest,
+            qualifiers={
+                "repository_url": repository_without_tag,
+                "tag": tag,
+            }
+        )
+        
+        component = {
+            "type": "container",
+            "name": repository_without_tag,
+            "purl": purl.to_string(),
+            "properties": [{
+                "name": property_name,
+                "value": index
```

Comment:
For base images, IMO the value can just be `"true"`

Also note that property values must be strings

Reply:
updated